talend-component-maven-plugin/src/it/web/.jenkins/Jenkinsfile

```diff
@@ -152,141 +166,118 @@ pipeline {
+          if ( user_name == null) { user_name = "ci-runner" }
+
+          println("Job info:\n" +
+              " - Tenant instance: ${tenantInstance}\n" +
+              " - Runtime version: ${runtimeVersion}\n" +
+              " - Test file executed: ${fileToRun}" + ".json\n" +
+              " - API Tester used environment: ${apiTesterEnv}\n")
+
+          println("Talend Test Orchestrator info:\n" +
+              "PLATFORM: ${params.PLATFORM}\n" +
+              "ENVIRONMENT: ${params.ENVIRONMENT}\n" +
+              "FRAGMENT_URL: ${params.FRAGMENT_URL}\n" +
+              "DEPLOYMENT_UNIT: ${params.DEPLOYMENT_UNIT}\n" +
+              "CYCLE: ${params.CYCLE}\n" +
+              "DEPLOYED_PRODUCTS: ${params.DEPLOYED_PRODUCTS}")
+
+          // updating build displayName
+          currentBuild.displayName = (
+              "#" + currentBuild.number + ": " + "${user_name} - " + runtimeVersion
+          )
+
+          // updating build description
+          currentBuild.description = (
+              "User: " + user_name + " - " + "Runtime: " + runtimeVersion + "\n" +
+                  "Project: " + fileToRun + "\n"
+          )
+        }
       }
     }
     stage('Prepare Test') {
       steps {
-
+        withCredentials([nexusCredentials]) {
           script {
             sh """
-              printf "======== Build the connector\n"
-              chmod +x ${SCRIPTS_PATH}/pom-edit.sh
-              ${SCRIPTS_PATH}/pom-edit.sh ${runtimeVersion} ${CONNECTORS_PATH}
-
-              cd ${TESTED_CONNECTOR_PATH}
-              mvn clean install -U\
-                                -T3C\
-                                -Dspotless.apply.skip=true\
-                                -Dspotbugs.skip=true\
-                                -Dcheckstyle.skip=true\
-                                -Drat.skip=true\
-                                -DskipTests\
-                                -Dmaven.javadoc.skip=true\
-                                -Dinvoker.skip=true\
-                                -Dmaven.artifact.threads=25
-
-              printf "======== TCK component server start\n"
-              chmod +x ${SCRIPTS_PATH}/server-start.sh
-              ${SCRIPTS_PATH}/server-start.sh ${TESTED_CONNECTOR_PATH} ${serverLog} ${serverPort} ${serverTimeout} &
-    
-              chmod +x ${SCRIPTS_PATH}/server-check.sh
-              ${SCRIPTS_PATH}/server-check.sh ${serverPort}
-  
-              cat ${TESTED_CONNECTOR_PATH}/${serverLog}
+              if [[ ${runtimeVersion} == *"-SNAPSHOT" ]]; then
+                echo "# Build a snapshot version"
+                "${scripts_path}/tcomp-build.sh" "${workspace}/pom.xml"
+              fi
+              
+              "${scripts_path}/server-registry-init.sh" "${download_dir}" "${install_dir}" "${coverage_dir}" \
+                                                         "${runtimeVersion}" \
+                                                         "1.36.0" "azure-dls-gen2" \
```

Comment:
Is this hardcoded version of connector expected ?

Reply:
Add a TODO : Will be removed when tests will be updated with the demo connector

Reply:
Ticket created to remove this connector in futur
TCOMP-2253 move from "azure-dls-gen2" to local build of the demo connector